packages/fxa-graphql-api/src/gql/dto/input/sign-in.ts

```diff
@@ -11,7 +11,7 @@ export class SignInOptionsInput {
   public keys?: boolean;
 
   @Field({ nullable: true })
-  public service?: string;
+  public service!: string;
```

Comment:
This intended? I believe service is optional

Reply:
Confirmed that when you signup via `http://localhost:3030` is it not set.

Reply:
hmm the joi validation rule for it is not optional, which was why I changed it, but I see that it also allows an empty string, so that's confusing.  I'll change it back.  Thanks.